src/controller/python/chip-device-ctrl.py

```diff
@@ -635,6 +636,17 @@ def echo_alive(message):
     print(message)
     return message
 
+def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
```

Comment:
why don't we just wrap around the actual do_resolve API, that way any updates made to these functions in future, we don't have to maintain our code

Reply:
Overall, we are not wrapping around the `do_`  methods as these methods don't return any values and just print the output for command line. In our case, we are looking for output and send it back to the client app. Also, these methods do some validation as command line users can pass any values but in our case, we are not doing any validation as we already have datatype defined.